spec/factories/reviews.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :random_review, class: Review do
```

Comment:
Sufficient to call it 'Review'. As long as whoever uses this factory does not explicitly define its fields, it is a convention for those fields to be populated by arbitrary or random legal values.

Reply:
I did not use random numbers to limit the rating to be 0-3 stars, for preventing the default values from randomly failing. 
For negative tests, will specify numbers not in the rang.